src/Avalonia.Base/Input/Platform/IPlatformDragSource.cs

```diff
@@ -1,11 +1,21 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 using Avalonia.Metadata;
 
 namespace Avalonia.Input.Platform
 {
     [Unstable]
     public interface IPlatformDragSource
```

Comment:
Should it be NonClientImplementable? 
This change is technically a binary breaking change. It wouldn't be one if it was NonClientImplementable.
Since likely nobody tried to implement this method, shouldn't be too late to change

Reply:
Yes it should. I've changed it. Still technically a breaking change indeed.

(The distinction between `Unstable`, `NotClientImplementable` and `PrivateApi` is often very thin.)